cli/src/cli.rs

```diff
@@ -1165,12 +1165,48 @@ fn process_balance(
     }
 }
 
-fn process_confirm(rpc_client: &RpcClient, signature: &Signature) -> ProcessResult {
-    match rpc_client.get_signature_status(&signature) {
+fn process_confirm(
+    rpc_client: &RpcClient,
+    config: &CliConfig,
+    signature: &Signature,
+) -> ProcessResult {
+    match rpc_client.get_signature_status_with_commitment_and_history(
+        &signature,
+        CommitmentConfig::max(),
+        true,
+    ) {
         Ok(status) => {
             if let Some(result) = status {
                 match result {
-                    Ok(_) => Ok("Confirmed".to_string()),
+                    Ok(_) => {
+                        if config.verbose {
+                            match rpc_client.get_confirmed_transaction(
+                                signature,
+                                solana_transaction_status::TransactionEncoding::Binary,
+                            ) {
+                                Ok(confirmed_transaction) => {
+                                    println!("\nTransaction:");
+                                    crate::display::println_transaction(
```

Comment:
This seems like it would be a good candidate for Output::Json handling. Shifting `display::println_transaction()` to an impl of Display for `TransactionWithStatusMeta` in solana-transaction-status instead would enable us to avoid making ledger-tool dependent on solana-cli, and also make it easier to add `getConfirmedBlock` to the cli in the future (not sure if there's a need there, though).

Reply:
I didn't feel good about adding a dependency from `solana-transaction-status` into the vote/stake programs, and wasn't prepare to create another crate.  Seems like ledger-tool ought to get folded into `cli/` one day anyway

Reply:
Fair enough. I'll keep it on my radar to work into OutputFormat handling at that time, then.